src/app/icd/ICDManager.cpp

```diff
@@ -105,7 +112,14 @@ void ICDManager::UpdateOperationState(OperationalState state)
         }
         DeviceLayer::SystemLayer().StartTimer(System::Clock::Timeout(idleModeInterval), OnIdleModeDone, this);
 
-        CHIP_ERROR err = DeviceLayer::ConnectivityMgr().SetPollingInterval(GetSlowPollingInterval());
+        System::Clock::Milliseconds32 slowPollInterval = GetSlowPollingInterval();
+        // When in SIT mode, the slow poll interval cannot be greater than kICDSitModePollingThreshold
```

Comment:
There is no kICDSitModePollingThreshold in the code here.  Maybe:
```suggestion
        // When in SIT mode, the slow poll interval cannot be greater than the SIT mode polling threshold, per spec.
```